carbonmark/components/pages/Project/ProjectFilters/styles.ts

```diff
@@ -0,0 +1,71 @@
+import { css } from "@emotion/css";
+import breakpoints from "@klimadao/lib/theme/breakpoints";
+
+export const pillContainer = css`
+  position: relative;
+  gap: 0.4rem;
+  width: 100%;
+  display: flex;
+  overflow-x: auto;
+  grid-column: full;
+  align-items: center;
+  max-width: 168rem;
+  align-items: center;
+  justify-self: center;
+
+  &::-webkit-scrollbar {
+    display: none;
+  }
+
+  ${breakpoints.desktop} {
+    overflow-x: hidden;
+  }
+
+  .more-text {
+    display: none;
+
+    ${breakpoints.desktop} {
+      top: 0;
+      right: 0;
+      z-index: 1;
+      display: block;
+      width: 12rem;
+      height: 3.2rem;
+      position: absolute;
+      background: linear-gradient(
+        270deg,
+        #eeeff5 55.13%,
+        rgba(238, 239, 245, 0) 100%
+      );
+
+      & p {
+        display: flex;
+        cursor: pointer;
+        height: 3.2rem;
+        align-items: center;
+        justify-content: end;
+        color: var(--bright-blue);
+        text-transform: uppercase;
+      }
+    }
+  }
+`;
+
+export const pill = css`
+  gap: 0.8rem;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  white-space: nowrap;
+  padding: 0.8rem 1.6rem;
+  background: #8b8fae;
+  border-radius: 4rem;
+  color: white;
+  font-weight: 600;
+  font-size: 1.2rem;
+  line-height: 1.6rem;
+
+  & svg {
+    cursor: pointer;
+  }
+`;
```

Comment:
Is this a common component that we might use elsewhere in the site? If so we should probably extract it

Reply:
I haven't seen anywhere else in the figma files where a similar component was used, so didn't extract it for this reason. Will take another look though and double check and can move it out into it's own component.